__tests__/data/braze_output.json

```diff
@@ -507,8 +507,164 @@
       "userId": "mickey@disney.com"
     }
   ],
-  {
-    "statusCode": 400,
-    "message": "Message type is not supported"
-  }
+  [
+    {
+      "body": {
+        "FORM": {},
+        "JSON": {
+          "attributes": [
+            {
+              "_update_existing_only": false,
+              "city": "Disney",
+              "country": "USA",
+              "email": "mickey@disney.com",
+              "firstname": "Mickey",
+              "user_alias": {
+                "alias_label": "rudder_id",
+                "alias_name": "e6ab2c5e-2cda-44a9-a962-e2f67df78bca"
+              }
+            }
+          ],
+          "events": [
+            {
+              "_update_existing_only": false,
+              "name": "KM Order Completed",
```

Comment:
@ItsSudip can you please check the test cases once, The event name should be "order completed" only, otherwise `getPurchaseObjs` method will not be called anyway. Look at line no 277-281. Edit the test case accordingly.

Reply:
Updated testcases.